packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -183,6 +184,7 @@ class SnackBar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final bool removeAnimation = RendererBinding.instance.assistiveTechnologyEnabled;
```

Comment:
we should put this in the MediaQuery so you don't need to use the binding. Using the binding directly is bad for testability, among other things.

Reply:
Moved this field to MediaQueryData and updated tests, PTAL